lib/timers.js

```diff
@@ -226,122 +284,105 @@ function listOnTimeout(handle, now) {
     // Check if this loop iteration is too early for the next timer.
     // This happens if there are more timers scheduled for later in the list.
     if (diff < msecs) {
-      var timeRemaining = msecs - (TimerWrap.now() - timer._idleStart);
-      if (timeRemaining <= 0) {
-        timeRemaining = 1;
-      }
-      handle.start(timeRemaining);
+      list.expiry = timer._idleStart + msecs;
+      list.id = timerListId++;
```

Comment:
Is Id being used as an alternate solution for `Check if this loop iteration is too early for the next timer.`? 

(That doesn't make sense to me how that would work, since you can't have two lists of the same duration anyways or insert will break.)

Reply:
You could start a timer for 120s and in 60s start a timer for 60s. Both have the same expiry. (This matches what libuv does to keep ordering consistent.)